app/assets/javascripts/admin/dataset/dataset_upload_view.js

```diff
@@ -127,7 +127,12 @@ class DatasetUploadView extends React.PureComponent<Props, State> {
                         { pattern: /[0-9a-zA-Z_-]+$/ },
                         {
                           validator: async (_rule, value, callback) => {
-                            const reasons = await isDatasetNameValid(value);
+                            const reasons = await isDatasetNameValid({
+                              name: value,
+                              owningOrganization: this.props.activeUser
+                                ? this.props.activeUser.organization
+                                : "",
```

Comment:
I assume this request would fail with organization == "", right? Can you just throw an error if activeUser is undefined? I think it should never happen anyway (since you have to be logged in for this view). But if it should happen for some reason, we have at least a sensible error thrown here :)